docs/Passing-Devices-with-Clear-Containers.md

```diff
@@ -0,0 +1,66 @@
+# Using devices with Clear Containers
+
+Host devices can be passed to a container with Docker using --device option
+with docker run. An example of passing a device with docker is given below:
+```
+docker run -it --device=/dev/sdb debian
```

Comment:
Needs a `$ sudo ` prefix.

Reply:
I usually have the user added to docker group, but I'll change this for consistency

Reply:
Right, but we don't encourage that for users for potential security reasons.